src/Arcus.Messaging.Pumps.ServiceBus/KeyRotation/AzureServiceBusLocation.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using GuardNet;
+
+namespace Arcus.Messaging.Pumps.ServiceBus.KeyRotation
+{
+    /// <summary>
+    /// Represents an environment of a Azure Service Bus resource; where the Azure Service Bus is located.
+    /// </summary>
+    public class AzureServiceBusLocation
```

Comment:
Location/Environment is odd naming, what about instance or simple namespace?
```suggestion
    public class AzureServiceBusNamespace
```

Reply:
Ok!